src/app/campaigns/campaign.js

```diff
@@ -0,0 +1,27 @@
+angular.module('crowdFundApp').controller('campaignController', function($scope, $http, $stateParams,$sce) {
+  $http.get('http://localhost:3000/campaign/'+$stateParams.id+'.json').then(
+    function(success){
+      $scope.campaign=success.data
```

Comment:
1. Please add a space before and after the `=`
2. Please remove all the debugging lines (`console.log`)